.storybook/main.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+  stories: [
+    "../src/**/*.stories.mdx",
+    "../src/**/*.stories.@(js|jsx|ts|tsx)",
+  ],
+  addons: [
+    "@storybook/addon-links",
+    "@storybook/addon-essentials",
+    "@storybook/addon-interactions",
+    "@storybook/preset-create-react-app"
+  ],
+  framework: "@storybook/react",
+  core: {
+    builder: "@storybook/builder-webpack5"
+  },
```

Comment:
```suggestion
    '../src/**/*.stories.mdx',
    '../src/**/*.stories.@(js|jsx|ts|tsx)',
  ],
  addons: [
    '@storybook/addon-links',
    '@storybook/addon-essentials',
    '@storybook/addon-interactions',
    '@storybook/preset-create-react-app'
  ],
  framework: '@storybook/react',
  core: {
    builder: '@storybook/builder-webpack5'
  },
```
Can we be able to apply `'` instead of `"` here, as a I see that in the other file you're using with the single quote?

Reply:
Fixed. Also now I enable the eslint to lint this file. Seems previously it is disable for this file by default 👍